lib/network/CachedImage.js

```diff
@@ -7,10 +7,16 @@
  *
  * NOTE: Images can also be of type 'data:svg+xml`. This code also works
  *       for svg, but the mipmapping may not be necessary.
+ *
+ * @class CachedImage
```

Comment:
Thanks for understanding; I realize that my response was nitpicking to the extreme. However, I do feel that if we're going to do the `jsdoc` thing then we should at least do it right and in such a way that it minimizes the maintenance work.

> I agree that if they're redundant and do not serve a purpose for ts generation or jsdoc generation, they should be removed.

Thanks. It also depends on if the typescript API generation needs them or not. I hope your check on that helps clarify things.